### PR TITLE
Update twitter to X on firefox /new product images (Fixes #13987)

### LINKED ISSUE #13987: Update twitter to X on firefox /new product images
## Description

Firefox new tab now displays 'X' instead of 'Twitter', and we need to update the product screenshots to reflect that on the Firefox /new page.

- There are 3 screenshots that I can find, but we should check code to see if there are any more.
- Jessica M has already created the new screenshots. I'll include them below, but developer should reach out to make sure they get the latest version and they are optimized and web ready.
- Confirm text should say "twitter' and not "X"

screen-mr1-dark_cropped
screen-mr1-dark
screen-mr1

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Updates images on the `/firefox/new/` page so they show the X logo rather than Twitter.

## Issue / Bugzilla link

#13987

## Testing

http://localhost:8000/en-US/firefox/new/

### CODE REVIEW

**Review by reemhamz — APPROVED** [review]
r+ 🍜 

**Edit:** 
I looked over the PR again and I think I missed something worth a discussion. Possible r+wc

**Comment by alexgibson** [comment]
@reemhamz what did you notice?

**Review by reemhamz — COMMENTED** [review]
Ah, sorry. I placed a review comment, but it didn't submit! Hopefully you can see the comment now

File: media/img/firefox/new/desktop/firefox-browser-light-1200.png
[the remark this answers is not in the document]
Comment (alexgibson) [reply media/img/firefox/new/desktop/firefox-browser-light-1200.png:1]: @reemhamz ah, thanks yeah!

I had the same question too, but Dan confirmed it should remain as "Twitter" for now (which matches what's currently in product). So I think we're OK here for now. Thanks for checking!

**Pull request merged**